src/multirust-cli/rustup_mode.rs

```diff
@@ -374,3 +383,9 @@ fn self_uninstall(m: &ArgMatches) -> Result<()> {
 
     self_update::uninstall(no_prompt)
 }
+
+fn telemetry(cfg: &Cfg, m: &ArgMatches) -> Result<()> {
+    let telemetry_string = m.value_of("enabled").unwrap();
+    
+    cfg.set_telemetry(telemetry_string)
```

Comment:
Instead of passing a string to `set_telemetry` please convert this to a bool first. I realize you'll have to convert that bool back to something else to store in the telemetry file, but I'd rather have the types still.